subsys/bluetooth/host/Kconfig.gatt

```diff
@@ -82,6 +82,14 @@ config BT_GATT_AUTO_SEC_REQ
 	  commercially available central devices that react negatively to
 	  receiving a Security Request immediately after reconnection.
 
+config BT_ATT_RETRY_ON_SEC_ERR
```

Comment:
Should this really be ATT and not GATT? If it’s ATT then maybe move it closer to the other ATT options in the beginning of this file?

Reply:
I think Johan's point was to simply move this config to lines start at line ~26 instead of 85, so that we group the `config BT_ATT_*` configs :)